client/src/state/editor/sortBlockMutation.js

```diff
@@ -28,13 +28,13 @@ const config = {
         },
       },
       update: store => {
-        const variables = readBlocksConfig.options({ pageId }).variables;
+        const variables = readBlocksConfig.options({ areaId }).variables;
         const data = store.readQuery({ query: readBlocksQuery, variables });
 
         // Query returns a deeply nested object. Explicit reconstruction via spreads is too verbose.
         // This is an alternative, relatively efficient way to deep clone
         const newData = JSON.parse(JSON.stringify(data));
-        let { edges } = newData.readOnePage.ElementalAreaIfExists.Elements;
+        let { edges } = newData.readOnePage.Elements;
```

Comment:
Should this now be `readOneElementalArea`?

Reply:
That's why sort wasn't working! I missed one 😅